src/definitions.ts

```diff
@@ -8,11 +8,11 @@ export interface RadarPlugin {
   addListener(eventName: 'log', listenerFunc: (result: { message: string }) => void): Promise<PluginListenerHandle> & PluginListenerHandle;
   initialize(options: { publishableKey: string }): void;
   setLogLevel(options: { level: string }): void;
-  setUserId(options: { userId: string }): void;
+  setUserId(options: { userId?: string }): void;
```

Comment:
Looks required in the `Plugin.swift` and `RadarPlugin.java` to me. Should switch the former to `let userId = call.getString("userId") ?? ""` if we want it optional.

But do we? It's not in `radar-sdk-ios`.